app/mentions/server/methods/getUserMentionsByChannel.js

```diff
@@ -1,10 +1,11 @@
 import { Meteor } from 'meteor/meteor';
 import { check } from 'meteor/check';
 
-import { Rooms, Users, Messages } from '../../../models';
+import { Rooms, Users } from '../../../models';
+import { Message } from '../../../../server/sdk';
 
 Meteor.methods({
-	getUserMentionsByChannel({ roomId, options }) {
+	getUserMentionsByChannel({ roomId, queryOptions }) {
```

Comment:
Breaking change

Reply:
Explaining: If you change the signature of the method or API it will require the other places calling and other external integrations to change how they call, so it's a braking change and we try to not have that. If you add a new parameter and it's optional it's ok, if it's required you are doing a breaking change as well. Sing this is not a parameter name but a field name of an object it will reflect on places calling it. Since you didn't change the places calling it, it's not only a breaking change, but it's breaking even the internal implementation.

You can use a rename of destructor parameter like:
```
getUserMentionsByChannel({ roomId, options: queryOptions }) {
```

Reply:
reverted